sns/parity_math.c

```diff
@@ -318,11 +322,7 @@ M0_INTERNAL int m0_parity_math_init(struct m0_parity_math *math,
 		math->pmi_parity_algo = M0_PARITY_CAL_ALGO_XOR;
 		return 0;
 	} else {
-		math->pmi_parity_algo = M0_PARITY_CAL_ALGO_REED_SOLOMON;
-		/*
-                 * init galois, only first call makes initialisation,
-		 * no de-initialisation needed.
-		 */
+		math->pmi_parity_algo = M0_PARITY_CAL_ALGO_ISAL;
 
```

Comment:
If we are going to replace Galois/REED_SOLOMON algorithm, are these data structures still needed?